solutionbox/structured_data/datalab_structured_data/_package.py

```diff
@@ -302,12 +329,27 @@ def local_train(train_dataset,
     for i in range(len(layer_sizes)):
       args.append('--layer_size%s=%s' % (i+1, str(layer_sizes[i])))
 
-  stderr = sys.stderr
-  sys.stderr = sys.stdout
-  print('Starting local training.')
-  trainer.task.main(args)
+
+  p = subprocess.Popen(' '.join(args),
+                       shell=True,
+                       stdout=subprocess.PIPE,
+                       stderr=subprocess.STDOUT)
+  print('Started local training in subprocess %s' % str(p.pid))
```

Comment:
I like 'Starting local training.' better because the subprocess thing is internal implementation that user should not care.